src/rest-server/deploy/legacy_user_migrate.py

```diff
@@ -79,6 +93,52 @@ def secret_data_prepare(self, user_info):
 
         return post_data_dict
 
+    def secret_data_prepare_v2(self, user_info_item):
+        meta_dict = dict()
+        meta_dict['name'] = user_info_item['metadata']['name']
+
+        grouplist = []
+        if base64.b64decode(user_info_item['data']['admin']) == 'true':
+            grouplist.append(self.admin_group)
+        for vc_name in user_info_item['metadata']['virtualCluster'].split(','):
+            self.vc_set.add(vc_name)
+            grouplist.append(vc_name)
+
+        extension = {}
+        if user_info_item['data']['githubPAT'] != '':
+            extension['githubPAT'] = base64.b64decode(user_info_item['data']['githubPAT'])
+
+        user_dict = {
+            'username': user_info_item['data']['username'],
+            'password': user_info_item['data']['password'],
+            'email': '',
+            'grouplist': str(base64.b64encode(json.dumps(grouplist)), 'utf-8'),
+            'extension': str(base64.b64encode(json.dumps(extension)), 'utf-8'),
+        }
+
+        post_data_dict = {}
+        post_data_dict['metadata'] = meta_dict
+        post_data_dict['data'] = user_dict
+
+        return post_data_dict
+
+    def secret_data_prepare_v2_group(self, groupname):
```

Comment:
This function is create group for VC, can not see from api. Should be secret_data_prepare_vc_group?